runtime/src/main/scala/akka/grpc/scaladsl/GrpcExceptionHandler.scala

```diff
@@ -26,7 +26,8 @@ object GrpcExceptionHandler {
     case _: UnsupportedOperationException ⇒
       Future.successful(GrpcResponseHelpers.status(Status.UNIMPLEMENTED))
     case other ⇒
-      Future.failed(other)
+      other.printStackTrace()
```

Comment:
We'd want this to go in a logger, not print to stdout.

Reply:
Agreed.

In Akka HTTP, the exception handler is a `PartialFunction[Throwable, Route]`, so you can get the logger from inside the `Route`. Route is a concept that is defined outside of akka-http-core, though, so we don't have access to it here.

The alternative is to change `val default` to `def default(...)` and close over the logger passed in there. That is still less flexible than doing it in the `Route` (the exception handler doesn't have access to the request context), but is likely sufficient? /cc @jrudolph

Reply:
I have done it